CMakeLists.txt

```diff
@@ -175,6 +175,12 @@ include(kconfig)
 message(STATUS "PX4 config: ${PX4_CONFIG}")
 message(STATUS "PX4 platform: ${PX4_PLATFORM}")
 
+if($ENV{CLION_IDE})
+	# CLion automatically executes some compiler commands after configuring the
```

Comment:
Is it doing anything different than regular cmake configure? Wondering if there might be a more general fix.